mongo/filterstore.go

```diff
@@ -377,9 +377,9 @@ func (s *FilterStore) CreateFilterOutput(ctx context.Context, filter *models.Fil
 
 // GetFilterOutput returns a filter output resource
 func (s *FilterStore) GetFilterOutput(ctx context.Context, filterID string) (*models.Filter, error) {
-	// NOTE: previously was filter_id.
-	// This should be _id/id. Confirm during PR.
-	query := bson.M{"filter_id": filterID}
+	// we have to match either the filter_id (in CMD) or the id (in Cantabular)
```

Comment:
I think a slightly longer comment explicitly saying why it led to having to support in this way would be helpful for future context